content/guides/folders/single/create-lock.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
Might it be nice to put these under `folders/locks/create.md` etc?

Reply:
Might be overthinking this so please take this with a grain of salt.

Reply:
I'll keep it as is for now as I think the list length is pretty good right now. If we add another series of APIs for folder operations let's revisit this and compartmentalize the content a bit more.